When deleting media library plugin: "Error: Command failed: npm run strapi -- uninstall upload -d"

**Describe the bug**
Unable to delete Media Library plugin.  Errors out with `Error: Command failed: npm run strapi -- uninstall upload -d`

**Steps to reproduce the behavior**

1. Install Strapi with yarn `sudo -u www-data yarn create strapi-app myproject --quickstart`
2. Create an admin user at `localhost:1337`
2. Delete Media Library plugin in the webUI

**Expected behavior**
Media Library Plugin disappears, triumphant music, confetti.  

**Results**
Strapi shows a "restart taking longer than usual..." message, then reloads and runs as normal, but with Media Library plugin still installed

**Log**

```
[2020-08-23T16:25:52.509Z] info Uninstalling upload...
[2020-08-23T16:25:53.826Z] info File changed: /srv/myproject/package.json
[2020-08-23T16:26:10.189Z] info File changed: /srv/myproject/yarn.lock
[2020-08-23T16:27:48.731Z] error Error: Command failed: npm run strapi -- uninstall upload -d                                                         
- Uninstalling dependencies
✔ Uninstalling dependencies
- Deleting old files
✔ Deleting old files
- Rebuilding admin UI
Command failed: npm run build
Killed                                                                                                                                                
npm ERR! code ELIFECYCLE                                                                                                                              
npm ERR! errno 137
npm ERR! myproject@0.1.0 build: `strapi build`                                                                                                          
npm ERR! Exit status 137                                                                                                                              
npm ERR!
npm ERR! Failed at the myproject@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-08-23T16_27_48_563Z-debug.log


> myproject@0.1.0 build /srv/myproject
> strapi build

Building your admin UI with development configuration ...
ℹ Compiling Webpack

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! myproject@0.1.0 strapi: `strapi "uninstall" "upload" "-d"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the myproject@0.1.0 strapi script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-08-23T16_27_48_692Z-debug.log


> myproject@0.1.0 strapi /srv/myproject
> strapi "uninstall" "upload" "-d"

    at makeError (/srv/myproject/node_modules/execa/index.js:174:9)
    at /srv/myproject/node_modules/execa/index.js:278:16
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.uninstallPlugin (/srv/myproject/node_modules/strapi-admin/controllers/Admin.js:84:7)
    at async /srv/myproject/node_modules/strapi/lib/middlewares/router/utils/routerChecker.js:79:22
    at async /srv/myproject/node_modules/strapi-utils/lib/policy.js:68:5
    at async /srv/myproject/node_modules/strapi/lib/middlewares/xss/index.js:26:9
[2020-08-23T16:27:48.739Z] debug DELETE /admin/plugins/uninstall/upload (116259 ms) 400
```

**System**

- Node.js version: v12.18.3
- NPM version: 6.14.6
- Yarn version: 1.22.4
- Strapi version: 3.1.4
- Database: development
- Operating system: Debian 10 Buster

--- a/packages/strapi-plugin-upload/package.json
+++ b/packages/strapi-plugin-upload/package.json
@@ -5,7 +5,8 @@
   "strapi": {
     "name": "Media Library",
     "icon": "cloud-upload-alt",
-    "description": "upload.plugin.description"
+    "description": "upload.plugin.description",
+    "required": true
   },
   "scripts": {
     "test": "echo \"no tests yet\""

